src/Compilers/VisualBasic/Portable/SymbolDisplay/CustomSymbolDisplayFormatter.vb

```diff
@@ -124,5 +122,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Public Shared Function ShortNameWithTypeArgsAndContainingTypes(symbol As Symbol) As FormattedSymbol
             Return New FormattedSymbol(symbol, ShortWithTypeArgsAndContainingTypesFormat)
         End Function
+
+        Public Shared Function DefaultErrorFormat(symbol As Symbol) As FormattedSymbol
+            Return New FormattedSymbol(symbol, SymbolDisplayFormat.VisualBasicErrorMessageFormat)
+        End Function
+
+        Public Shared Function DefaultErrorFormatIfSpecialType(arg As Object) As Object
```

Comment:
We should inline this helper, there is only one call site for it in MissingMetadataTypeSymbol and the logic can be much simpler because we know we are dealing with TypeSymbol.